docs/RFCS/20160730_streaming_snapshots.md

```diff
@@ -1,5 +1,5 @@
 - Feature Name: streaming_snapshots
-- Status: in-progress
+- Status: completed
```

Comment:
I think this might be still in-progress - the receiver doesn't stream the snapshot yet.

Reply:
You're right, thanks. Changed back.